### PR TITLE
#167190422 setup postgres database

### PR DESCRIPTION
**Pull request by obayomi96**
#### What does this PR do?
- Sets up Postgres as database engine for the App

#### Description of Task to be completed?

- Install pg, pg-hstore, sequelize-cli, and sequelize
- Set up configuration
- Set up migrations
- Set up models
- Connect with the database

#### How should this be manually tested?

- clone the repo

 - Run the following commands

`npm install`
`npm run migrate`

#### Any background context you want to provide?

- Ensure to add your DATABASE_URL and TEST_DATABASE_URL to the .env file as specified in the .env.sample file

#### What are the relevant pivotal tracker stories?

#167190422

#### Screenshots (if appropriate)

Screenshot 2019-07-24 at 1 16 31 PM

### CODE REVIEW

**Review by encodedBicoding — APPROVED** [review]
LGTM

**Review by cvjude — APPROVED** [review]
LGTM

**Review by Hadeneekeh — APPROVED** [review]

**Review by Lundii — COMMENTED** [review]
A PR was just merged. Rebase your branch

**Comment by obayomi96** [comment]
> A PR was just merged. Rebase your branch

Yes, I've done that. Thanks

**Review by chiboycalix — APPROVED** [review]
Nice work @obayomi96 , the code looks very clean.

**Review by Lundii — APPROVED** [review]
Works fine on my Local. Good job

File: server/db/models/user.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
```
Comment (chuxmykel) [inline server/db/models/user.js]: Great Job @obayomi96. I just tested the setup and it works really well.

**Review by chuxmykel — APPROVED** [review]
Great Job @obayomi96. I just ran the migration and it works well.

**Review by topseySuave — APPROVED** [review]

**Pull request merged**